graphql/mutation_resolver.go

```diff
@@ -852,7 +852,15 @@ func (r *mutationResolver) RemoveVolume(ctx context.Context, volumeID string) (b
 }
 
 // UpdateSpawnHostStatus is the resolver for the updateSpawnHostStatus field.
-func (r *mutationResolver) UpdateSpawnHostStatus(ctx context.Context, hostID string, action SpawnHostStatusActions) (*restModel.APIHost, error) {
+func (r *mutationResolver) UpdateSpawnHostStatus(ctx context.Context, hostID string, action SpawnHostStatusActions, updateSpawnHostStatusInput *UpdateSpawnHostStatusInput) (*restModel.APIHost, error) {
+	shouldKeepOff := false
+	// TODO: Use input object throughout resolver once deprecated fields are removed
+	if updateSpawnHostStatusInput != nil {
+		hostID = updateSpawnHostStatusInput.HostID
+		action = updateSpawnHostStatusInput.Action
+		shouldKeepOff = *updateSpawnHostStatusInput.ShouldKeepOff
```

Comment:
I don't know how GraphQL handles bool pointers, but is this always set? Alternatively, would it make sense to use `utility.FromBoolPtr` to dereference it safely in case it's nil?